.github/CODEOWNERS

```
@@ -671,3 +671,4 @@ pep-8016.rst  @njsmith @dstufft
 pep-8100.rst  @njsmith
 # pep-8101.rst
 # pep-8102.rst
+pep-9999.rst @thejcannon @ericvsmith
```

Only members of `@python` can be listed in CODEOWNERS due to GH limitations:

```suggestion
pep-0712.rst @ericvsmith
```